### ISSUE TITLE
[1.4.0.009] Crash when using the ritual "recall to gateway" to tp to a gateway in a other dimension

### ISSUE THREAD

**Issue by N7shadow2000** [Mithion/ArsMagica2#1527]
Here the crash report 
http://pastebin.com/D38D4EBt
Forge Version: 10.13.4.1614 - 1.7.10
Error type:    java.lang.StackOverflowError: Ticking entity
I created a gateway in the nether and tried to use the Ritual "Recall to Gateway" in the overworld to tp to the gateway in the nether with its rune combination

**Comment by ThatOneTraitor** [comment]
Try doing it in a new world.

**Comment by N7shadow2000** [comment]
I have found a few conditions for this bug to trigger but first the way to achive this bug:

1: build a gateway in the nether or overworld and give it a rune combination.
2: build a gateway in another dimension and give it different rune combination.
3: use the ritual "recall to gateway" using the rune combination of the gate in the other dimension.
4: now every entity in your world should be frozen (crash)

Condition: You need 2 gateways in different dimensions. If you now use the ritual to tp to the gateway in the other dimension while having the other gateway in your current dimension your game will crash

**Comment by AterIgnis** [comment]
I am not sure if gateways/recall actually work across dimensions, but if they do not, then this may be related to my issue #1531.

**Comment by ThatOneTraitor** [comment]
Yeah i noticed that there as a line of code that kept on looping:

`  at am2.spell.shapes.Self.beginStackStage(Self.java:25)`

**Comment by jjtParadox** [comment]
Dimension transfers have always been wonky and hard to do, but hopefully a fix can be applied here similar to the fix for divine and ender intervention.
